packages/datetime/src/dateInput.tsx

```diff
@@ -7,7 +7,7 @@
 import * as classNames from "classnames";
 import * as moment from "moment";
 import * as React from "react";
-import * as ReactDayPicker from "react-day-picker";
+import { DayPickerProps } from "react-day-picker/types/props";
```

Comment:
is it possible to import this without a submodule? like from `react-day-picker` itself like it used to be? (i haven't poked at the actual package yet)

Reply:
on further examination, it seems not. 👍

Reply:
in this case, @Skovy please add the tslint-config exception as suggested by @adidahiya above.

Reply:
👍 yea, I was looking into that as well and was disappointed only `DayPicker` was exported.

Reply:
Filed: https://github.com/gpbl/react-day-picker/issues/617